examples/nirfmxspecan/acp-basic.py

```diff
@@ -0,0 +1,322 @@
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgPowerUnits(
+            nirfmxspecan_types.ACPCfgPowerUnitsRequest(
+                instrument=None,
+                selector_string="",
+                power_units=nirfmxspecan_types.ACP_POWER_UNITS_DBM,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgAveraging(
+            nirfmxspecan_types.ACPCfgAveragingRequest(
+                instrument=None,
+                selector_string="",
+                averaging_enabled=nirfmxspecan_types.ACP_AVERAGING_ENABLED_FALSE,
+                averaging_count=10,
+                averaging_type=nirfmxspecan_types.ACP_AVERAGING_TYPE_RMS,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgRBWFilter(
+            nirfmxspecan_types.ACPCfgRBWFilterRequest(
+                instrument=None,
+                selector_string="",
+                rbw_auto=nirfmxspecan_types.ACP_RBW_AUTO_BANDWIDTH_TRUE,
+                rbw=10.0e3,
+                rbw_filter_type=nirfmxspecan_types.ACP_RBW_FILTER_TYPE_GAUSSIAN,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgSweepTime(
+            nirfmxspecan_types.ACPCfgSweepTimeRequest(
+                instrument=None,
+                selector_string="",
+                sweep_time_auto=nirfmxspecan_types.ACP_SWEEP_TIME_AUTO_TRUE,
+                sweep_time_interval=1.00e-3,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgNoiseCompensationEnabled(
+            nirfmxspecan_types.ACPCfgNoiseCompensationEnabledRequest(
+                instrument=None,
+                selector_string="",
+                noise_compensation_enabled=nirfmxspecan_types.ACP_NOISE_COMPENSATION_ENABLED_FALSE,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgCarrierIntegrationBandwidth(
+            nirfmxspecan_types.ACPCfgCarrierIntegrationBandwidthRequest(
+                instrument=None, selector_string="", integration_bandwidth=1.0e6
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgCarrierRRCFilter(
+            nirfmxspecan_types.ACPCfgCarrierRRCFilterRequest(
+                instrument=None,
+                selector_string="",
+                rrc_filter_enabled=nirfmxspecan_types.ACP_CARRIER_RRC_FILTER_ENABLED_FALSE,
+                rrc_alpha=0.220,
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgNumberOfOffsets(
+            nirfmxspecan_types.ACPCfgNumberOfOffsetsRequest(
+                instrument=None, selector_string="", number_of_offsets=NUMBER_OF_OFFSETS
+            )
+        )
+    )
+    raise_if_error(
+        client.ACPCfgOffsetArray(
+            nirfmxspecan_types.ACPCfgOffsetArrayRequest(
+                instrument=None,
+                selector_string="",
+                offset_frequency=[1.0e6, 2.0e6],
+                offset_sideband=None,
+                offset_enabled=None,
```

Comment:
This call (mirrored from the C example, which passes in NULL for the last two parameters) errors with:
"The sizes of linked repeated fields [offsetFrequency, offsetSideband, offsetEnabled] do not match"
Filed [AB#1836020](https://ni.visualstudio.com/DevCentral/_workitems/edit/1836020).
(But now I'm going to remove this offending example, since I just realized I copied from the Acp example instead of the AcpBasic example.)
(FYI: @astarche)

Reply:
I think we want to use this mechanism: #456. Added a comment to the bug. We'll have to double-check the docs that it's really implemented the same way.

These should not be `None`, by the way. In `protobuf` the arrays for repeated fields can be empty but not null, so it's confusing to appear to be passing `None` (granted maybe it does the same thing). Either passing `[]` or not passing the argument at all would be fine. I don't think there's ever a case where we want to pass `None`/null to a protobuf/grpc stub.